### PR TITLE
feat(route): add 36kr资讯热榜

### LINKED ISSUE #11887: 36氪
### 类型

新媒体

### 网站地址

https://36kr.com/

### 网站描述

中国知名商业网

### 需要生成什么内容？

24小时热榜：

image

https://36kr.com/hot-list/catalog
人气榜：
综合榜：
收藏榜：

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved Issue

Close #11887

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/36kr/hot-list
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/36kr/hot-list.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    const data = JSON.parse(response.data.match(new RegExp('"' + categories[category].regex + '":(\\[.*?\\])'))[1]);
+
+    let items = data
+        .slice(0, ctx.query.limit ? parseInt(ctx.query.limit) : 30)
+        .filter((item) => item.itemType !== 0)
+        .map((item) => {
+            item = item.templateMaterial ?? item;
+            return {
+                title: item.widgetTitle.replace(/<\/?em>/g, ''),
+                author: item.author,
```
Comment (TonyRL) [inline lib/v2/36kr/hot-list.js:45]:
```suggestion
                author: item.authorName,
```

File: lib/v2/36kr/hot-list.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                pubDate: parseDate(item.publishTime),
+                link: `${rootUrl}/p/${item.itemId}`,
+                description: item.widgetContent ?? item.content,
```
Comment (TonyRL) [inline lib/v2/36kr/hot-list.js:48]:
```suggestion
                description: item.summary,
```
I can't find `item.author`, `item.widgetContent` or `item.content`.

File: lib/v2/36kr/hot-list.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                    ).articleDetail.articleDetailData.data;
+                    item.description = content.widgetContent;
+                } else {
+                    const $ = cheerio.load(detailResponse.body);
+                    item.description = $('div.articleDetailContent').html();
+                }
+                return item;
+            })
+        )
+    );
```
Comment (TonyRL) [inline lib/v2/36kr/hot-list.js:79]: Consider combining this with 
https://github.com/DIYgod/RSSHub/blob/3c6be9309b85de53a297ad78936b476239300b92/lib/v2/36kr/index.js#L47-L70

File: docs/new-media.md [hunk 4]
```diff
@@ -68,6 +68,16 @@ pageClass: routes
 
 <Route author="nczitzk" example="/36kr/search/newsflashes/ofo" path="/36kr/search/newsflashes/:keyword" :paramsDesc="['关键字']" />
 
+### 资讯热榜
+
+<Route author="nczitzk" example="/36kr/hot-list" path="/36kr/hot-list/:category?" :paramsDesc="['分类，默认为资讯人气榜']">
```
[the remark this answers is not in the document]
Comment (nczitzk) [reply docs/new-media.md:73]: 😅

File: lib/v2/36kr/utils.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const CryptoJS = require('crypto-js');
+
+const rootUrl = 'https://www.36kr.com';
+
+module.exports = {
+    rootUrl,
+    ProcessItem: async (item, tryGet) =>
```
[the remark this answers is not in the document]
Comment (nczitzk) [reply lib/v2/36kr/utils.js:9]: 👌

**Pull request merged**